pipeline.yaml

```diff
@@ -61,6 +61,9 @@ steps:
     # Indicates whether or not a model that fails to meet performance thresholds should still
     # be registered to the MLflow Model Registry
     allow_non_validated_model: true
+    # Set the registry server URI. This property is especially useful if you have a registry
+    # server that’s different from the tracking server.
+    # registry_uri: {{REGISTRY_URI}}
```

Comment:
Registry URI should be part of infrastructure instead of register step. I'd prefer to have it defined exclusively in profiles. See recommendation below.